### ISSUE TITLE
HACS integration

### ISSUE THREAD

**Issue by user897943** [betaboon/philips-airpurifier#16]
The current integration on HACS does not work for 2889/10.

Could this please be integrated on HACS for easier installation and updating?

**Comment by kongo09** [comment]
What exactly do you mean with "does not work"? I do have a 2889/10 setup via HACS
![image](https://user-images.githubusercontent.com/640121/108506387-faab2080-72b8-11eb-86af-7504f80c10c0.png)
![image](https://user-images.githubusercontent.com/640121/108506441-131b3b00-72b9-11eb-8466-cd0958e60fe9.png)

**Comment by user897943** [comment]
Sorry, I mean the "default" one on HACS: xMrVizzy / philips-airpurifier does not work with the 2889/10.

I can see that you have this on HACS, could you please tell me how to add the repo? What URL do you use? I will try here. Thanks in advance

**Comment by user897943** [comment]
I added this to HACS, if someone's wondering how here's how I did it:

Open HACS, click on Integrations, on the top click on the three dots then click on custom repositories then paste the following URL into the Add custom repository URL field: https://github.com/betaboon/philips-airpurifier now select Integrations on the category dropdown and click add. Restart home assistant, configure the configuration.yaml. It works perfectly, I can now see and control my air purifiers, thanks to the maintainers of this integration!
